build-script-helper.py

```diff
@@ -261,7 +261,7 @@ def docc_bin_path(swift_exec, args, env, verbose):
   if verbose:
     print(' '.join([escape_cmd_arg(arg) for arg in cmd]))
   return os.path.join(
-    subprocess.check_output(cmd, env=env).strip(), 'docc')
+    subprocess.check_output(cmd, env=env).strip().decode(), 'docc')
```

Comment:
Should we be decoding the the bytes to a string before calling `strip()`?
```suggestion
    subprocess.check_output(cmd, env=env).decode().strip(), 'docc')
```

Reply:
I don't believe so; that was the only spot I found where it was trying to join a byte array and a string.